src/App.js

```diff
@@ -489,6 +490,10 @@ export const App = ({
               path={PATHS.DASHBOARDS}
               render={(props) => <LoadableDashboardsPage {...props} />}
             />
+            <Route
+              path={`${PATHS.DASHBOARDS}-old`}
+              render={(props) => <LoadableOldDashboardsPage isDesktop={isDesktop} {...props} />}
+            />
```

Comment:
What is the reason for keeping the old route for production? It will not be maintained

Reply:
It's not the final PR, so there are some duplicates and old route
Also updating old components to new styles